user-org-util/src/main/java/org/sunbird/util/ProjectLogger.java

```diff
@@ -164,4 +164,54 @@ private static String getBELog(
     }
     return jsonMessage;
   }
+  /**
+   * This method will be called from Application class(default startup class). which will
+   * automatically set the Log level by taking its value from system environment file.
+   */
+  public static void setUserOrgServiceProjectLogger() {
+
+    String level = getLevelFromEnv();
+    switch (level.toUpperCase()) {
+      case "INFO":
+        changeLogLevel(Level.INFO);
+        break;
+      case "ERROR":
+        changeLogLevel(Level.ERROR);
+        break;
+      case "DEBUG":
+        changeLogLevel(Level.DEBUG);
+        break;
+      case "WARN":
+        changeLogLevel(Level.WARN);
+        break;
+      case "ALL":
+        changeLogLevel(Level.ALL);
+        break;
+      default:
+        changeLogLevel(Level.INFO);
+    }
+  }
+
+  /**
+   * This method will be take Log Level as input params and will set the Log Level dynamically.
+   * Configurator class is in package org.apache.logging.log4j.core.config.
+   *
+   * @param level
+   */
+  public static void changeLogLevel(Level level) {
+    Configurator.setLevel(rootLogger.getName(), level);
+    Configurator.setRootLevel(Level.INFO);
+    log(String.format("Log Level set to :%s", level), LoggerEnum.INFO.name());
+  }
+
+  /**
+   * This method will take value of Logger level from System environment.
+   *
+   * @return String value of log level if preset otherwise will return the default Logger Level INFO
+   */
+  public static String getLevelFromEnv() {
+    return StringUtils.isNotBlank(System.getenv(UserOrgJsonKey.USER_ORG_SERVICE_LOG_LEVEL))
```

Comment:
Use property reader class , we already have in sunbird repo. 
1. For any env first it will read from environment variable , if not present then it will read from properties file. We need to reuse same class.